### PR TITLE
Added type StockChange

### PR DESCRIPTION
**Pull request by Argee88**
Added type StockChange and fixed a few things...

### CODE REVIEW

File: src/Type/ShipmentOrdersGet.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Collmex ShipmentOrdersGet Type
+ *
+ * @author    Marcus Jaschen <mail@marcusjaschen.de>
+ * @author    René Galle <renegalle.webdevelopment@gmail.com>
+ * @license   http://www.opensource.org/licenses/mit-license MIT License
+ * @link      https://github.com/mjaschen/collmex
+ */
```
Comment (oliverklee) [inline src/Type/ShipmentOrdersGet.php]: Please let's not add new file dock blocks. (I've removed the existing ones in #81.) Thanks!

Comment (Argee88) [reply src/Type/ShipmentOrdersGet.php]: But the property stuff is to be kept right? :D

Comment (oliverklee) [reply src/Type/ShipmentOrdersGet.php]: Yes, the *class* doc block should stay. Only the (redundant) *file* doc block should go away.

Comment (Argee88) [reply src/Type/ShipmentOrdersGet.php]: Got it^^ I will adjust it and push the commit as soon as I‘m on the pc again ;)

**Comment by mjaschen** [comment]
Hi,

thanks for the PR!

(I reverted the the commit [fbbdbdc](https://github.com/mjaschen/collmex/pull/95/commits/fbbdbdc32f2ef8a25ef3ddb7080e8e6a4541ca53) because it breaks backwards compatibility just for cosmetic reasons.)

**Pull request closed without merging**